src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -22,6 +22,15 @@
           </i>
           <ng-container i18n>Set Cluster-wide OSD Flags</ng-container>
         </button>
+
+        <button class="btn btn-sm btn-default btn-label tc_configureCluster"
+                type="button"
+                (click)="configureQosParamsAction()">
+          <i class="fa fa-fw fa-cog"
+             aria-hidden="true">
+          </i>
+          <ng-container i18n>Set OSD recovery speed</ng-container>
```

Comment:
I'm asking myself if it is necessary to include the word ``OSD`` here. I think the user is aware that it refers to OSD's because he is in the appropriate tab. The action button and the two additional buttons require many space, removing ``OSD`` will reduce this a little bit. If you remove this please also modify the other button text.

![auswahl_001](https://user-images.githubusercontent.com/1897962/48135705-f8665900-e29d-11e8-89de-8fe65ce389c5.png)

Reply:
You mean I should also replace 'Set Cluster-wide OSD Flags' by 'Set Cluster-wide Flags', right? - If yes I fully agree.

Reply:
Just wondering if it makes sense to also use "Cluster-wide" within the name of the "Set recovery speed"-button (for the sake of consistency and to make it more clear that it'll affect all OSDs when setting a profile)?

Reply:
@callithea yes, sure I can add it :)